packages/composables/src/useWishlist.ts

```diff
@@ -23,9 +28,21 @@ export type UseWishlistReturn = {
    */
   items: ComputedRef<string[]>;
   /**
-   * Wishlist items count
+   * Current page number
+   */
+  getCurrentPage(): ComputedRef<number>;
```

Comment:
same here - should be only a computed property, not a function
```suggestion
  currentPage: ComputedRef<number>;
```